internal/acceptance/sql_table_test.go

```diff
@@ -0,0 +1,191 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestAccResourceSqlTable_Managed(t *testing.T) {
```

Comment:
acceptance test convention

```suggestion
func TestUcAccResourceSqlTable_Managed(t *testing.T) {
```